src/Sylius/Bundle/ApiBundle/DataTransformer/RemoveItemFromCartDataTransformer.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\DataTransformer;
+
+use ApiPlatform\Core\DataTransformer\DataTransformerInterface;
+use Sylius\Bundle\ApiBundle\Command\Cart\RemoveItemFromCart;
+use Sylius\Component\Core\Model\OrderInterface;
+use Webmozart\Assert\Assert;
+
+final class RemoveItemFromCartDataTransformer implements DataTransformerInterface
```

Comment:
This is a one-to-one copy-paste of `AddItemToCartInputDataTransformer`, am I right? IF so, maybe we should provide some generic data transformer, which will enrich all token aware commands?

Reply:
I will improve it in next PR